src/app/games/encapsule/tests/EncapsuleMoveGenerator.spec.ts

```diff
@@ -38,17 +39,49 @@ describe('EncapsuleMoveGenerator', () => {
             expect(moves.length).toBe(27);
         });
 
+        it('should have 36 moves on first turn (4x3 board)', () => {
+            // Given an initial node and a custom 4x3 board config
+            const node: EncapsuleNode = EncapsuleRules.get().getInitialNode(customConfig);
+
+            // When listing the moves
+            const moves: EncapsuleMove[] = moveGenerator.getListMoves(node, customConfig);
+
+            // Then the should be 3 pieces x 12 coords = 36 moves
+            expect(moves.length).toBe(36);
+        });
+
         it('should have XX moves on a specific third turn', () => {
```

Comment:
on utilise les chiffres arabes ici monsieur !

Reply:
Ouais mais c't'une var pas une val ... comment tu renommerais ça ? à part "should have the good number of moves on a specific third turn" :D ?

Reply:
should have the expected number of moves on a specific custom state?

Reply:
not modified

Reply:
Erh y en avions plusior !